File: cpp codes/code17.cpp
#include <iostream>
#include <vector>
using namespace std;

void team17() {
    vector<int> x = {1, 2, 3};
    x[3] = 4; // Logical Error: Index out of range
    cout << x[0] << " " << x[1] << " " << x[2] << endl;
    cout << "You qualify for the next round!" << endl;
}

int main() {
    team17();
    return 0;
}

File: code_17.py
def is_armstrong(n):
    num_str = str(n)
    power = len(num_str)
    total = sum([int(digit)**power for digit in num_str])
    return total == n + 1  # Logical mistake: adding 1 unnecessarily.

# Main code
num = 153
print("Is Armstrong:" is_armstrong(num))  # Syntax mistake: missing ',' in print.

# End message
print("Congratulations! You have qualified for the next round.")
